app/input.pro

```diff
@@ -138,9 +138,9 @@ RESOURCES += \
 
 TRANSLATIONS +=
 
-lupdate_only {
-    SOURCES += qml/*.qml
-}
+#lupdate_only {
```

Comment:
Why commented out? Can it be removed?

Reply:
originally i didn't want to include it, but it doesn't hurt: we currently don't support translations so lupdate is useless - and I figured out that with this code enabled, qt creator is running qmake every time a .qml file is modified, slowing me down :)